src/hex2nix.erl

```diff
@@ -119,6 +119,21 @@ do_main(Opts) ->
             ok
     end.
 
+-spec remove_unbuildable_deps([h2n_fetcher:dep_desc()]) -> [h2n_fetcher:dep_desc()].
+remove_unbuildable_deps(Deps) ->
+    lists:filter(fun unbuildable_dep/1, Deps).
+
+-spec unbuildable_dep(h2n_fetcher:dep_desc()) -> boolean().
+unbuildable_dep(#dep_desc{deps = Deps,
```

Comment:
Is this named backwards? I think maybe you intended to call it "buildable_dep" because it looks like it returns "false" if it's unbuildable not "true"